sunpy/data/test/__init__.py

```diff
@@ -41,26 +42,45 @@ def get_test_filepath(filename, **kwargs):
     """
     return get_pkg_data_filename(filename, package="sunpy.data.test", **kwargs)
 
+
 def get_available_test_data():
     """
     Prints all available test data
 
     """
-    file_name=""
-    for fn in get_pkg_data_filenames('test', 'sunpy.data','*'):
-        if(os.path.isdir(fn)):
-            for dir_contents in os.listdir(fn):
-                print (dir_contents)
-        else:
-            for file_path in reversed(fn):
-                if(file_path == '/'):
-                    if(file_name == "__init__.py" or file_name == "bug.py"):
-                        break
+    file_name = ""
+    if(os.name == 'nt'):
```

Comment:
This is breaking the  [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) principle. You shouldn't need to repeat the whole thing again only for one line where the slashes are not the same. Check out `os.sep` or [`os.path`](https://docs.python.org/3/library/os.path.html) for some things that may help you.